include/stronk/cmath.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <cmath>
+
+#include "stronk/stronk.hpp"
+#include "stronk/unit.hpp"
+
+namespace twig
+{
+
+template<stronk_like StronkT>
+    requires(!unit_value_like<StronkT>)  // units should divide by T(1) first
+auto log(StronkT value) -> decltype(log(value.template unwrap<StronkT>()))
```

Comment:
The existing math functions are exposed both as a skill with a method and a global function.

I think we should either keep on adding that, or remove it altogether.
What do you think?

Reply:
I just want to have it as global function